src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,26 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  gap: 48px;
```

Comment:
You may want to add something to center the content
for example margin: 0 auto;